### DISCUSSION TITLE
D-bus paths

### DISCUSSION THREAD

**Discussion by stanhausc** [Louisvdw/dbus-serialbattery#542] [General]
I would like to create a new path, for example, take a value from my BMS and publish it in the Cerbo GX interface and a qml file that is also new. I've been researching these references [1](https://github.com/victronenergy/venus/wiki/dbus#battery), [2](https://github.com/victronenergy/venus/wiki/dbus-api). But I haven't been successful. Does anyone know which files are related to create a new path?

**Comment by mr-manuel** [comment]
Yes, what do you want to do exactly? Do you want extend this driver or create a new one?

**Reply by stanhausc** [reply]
I use this drive, but I wanted to create a new QML file, to view data from another separate BMS, if I manage to create a new path in dbus, I just create a new qml that will appear on Cerbo's screen.
For example, I receive and unpack the data received from the BMS, and "self.soc = data" is enough for the drive to publish on the dbus, I wanted to make a self.soc2 to publish the soc of the second bms.

**Reply by mr-manuel** [reply]
Have a look at this PR: https://github.com/Louisvdw/dbus-serialbattery/pull/440/files
